pkgs/misc/emulators/caprice32/default.nix

```
@@ -1,29 +1,47 @@
-{ stdenv, fetchFromGitHub, libpng, pkgconfig, SDL, freetype, zlib }:
+{ stdenv, fetchFromGitHub, desktop-file-utils, libpng
+, pkgconfig, SDL, freetype, zlib }:
 
 stdenv.mkDerivation rec {
 
   pname = "caprice32";
-  version = "4.5.0";
+  version = "4.6.0";
 
   src = fetchFromGitHub {
     repo = "caprice32";
     rev = "v${version}";
     owner = "ColinPitrat";
-    sha256 = "056vrf5yq1574g93ix8hnjqqbdqza3qcjv0f8rvpsslqcbizma9y";
+    sha256 = "0hng5krwgc1h9bz1xlkp2hwnvas965nd7sb3z9mb2m6x9ghxlacz";
   };
 
-  postPatch = "substituteInPlace cap32.cfg --replace /usr/local $out";
-  nativeBuildInputs = [ pkgconfig ];
+  nativeBuildInputs = [ desktop-file-utils pkgconfig ];
   buildInputs = [ libpng SDL freetype zlib ];
 
-  #fix GIT_HASH avoid depend on git 
-  makeFlags = [ "GIT_HASH=${src.rev}" "DESTDIR=$(out)" "prefix=/"];
+  makeFlags = [
+    "APP_PATH=${placeholder ''out''}/share/caprice32/"
+    "RELEASE=1"
+    "DESTDIR=${placeholder ''out''}"
```

These can just be normal strings.

```suggestion
    "DESTDIR=${placeholder "out"}"
```